### DISCUSSION TITLE
podman having issues pulling from any repository on this specific library

### DISCUSSION THREAD

**Discussion by allangdtl** [containers/podman#24162] [Q&A]
I am currently testing podman to possibly replace docker using binaries on Ubuntu 20.04:
https://github.com/mgoltzsche/podman-static

✔ docker.io/library/quasar_frontend:latest
Trying to pull docker.io/library/quasar_frontend:latest...
Error: initializing source docker://quasar_frontend:latest: reading manifest latest in docker.io/library/quasar_frontend: requested access to the resource is denied

Other libraries are okay:
Resolved "postgres" as an alias (/home/allang/.cache/containers/short-name-aliases.conf)
Trying to pull docker.io/library/postgres:13...
Getting image source signatures
Copying blob bb412209e710 done   |
Copying blob f7980a65a9a8 done   |
Copying blob b6b8875f339b done   |
Copying blob 302e3ee49805 done   |
Copying blob 7590b342caf0 done   |
Copying blob f82559c6448e done   |
Copying blob 6f5d8ce8b2b8 done   |
Copying blob 3b70089fd7f0 done   |
Copying blob cd62cb0e4a5c done   |
Copying blob ea723a51eb63 done   |
Copying blob a65d8590c845 done   |
Copying blob aaa0d453b822 done   |
Copying blob 405161c5b60a done   |
Copying blob cf5ab8bd4d28 done   |
Copying config d76feacfc4 done   |
Writing manifest to image destination
9e74ebf990106954072bdcc4b2d2d6acaf64627857d37053933b72b32f2e7f18

Resolved "redis" as an alias (/home/allang/.cache/containers/short-name-aliases.conf)
Trying to pull docker.io/library/redis:latest...
Getting image source signatures
Copying blob 302e3ee49805 skipped: already exists
Copying blob 96377887d476 done   |
Copying blob 5d0249d9189d done   |
Copying blob 4825c5e95815 done   |
Copying blob b0ce50685fa2 done   |
Copying blob 455886c7d31b done   |
Copying blob 4f4fb700ef54 done   |
Copying blob 5fac73c23c9b done   |
Copying config 7e49ed81b4 done   |
Writing manifest to image destination
311f21c6af10b23415ae001ec756b1dc23b81c1eef48d57fd5e5097ed01d4cad

**Comment by allangdtl** [comment]
I have both docker on another machine and it runs fine.

**Comment by rhatdan** [comment]
Looks like you have an authorization issue.

**Comment by allangdtl** [comment]
I am able to "podman login" to our local registry:
`Authenticating with existing credentials for <registry>:5000
Existing credentials are valid. Already logged in to <registry>:5000`

I can also pull from other repositories:
`Trying to pull docker.io/library/redis:latest...
Getting image source signatures
Copying blob 302e3ee49805 skipped: already exists
Copying blob 96377887d476 done   |
Copying blob 5d0249d9189d done   |
Copying blob 4825c5e95815 done   |
Copying blob b0ce50685fa2 done   |
Copying blob 455886c7d31b done   |
Copying blob 4f4fb700ef54 done   |
Copying blob 5fac73c23c9b done   |
Copying config 7e49ed81b4 done   |
Writing manifest to image destination`

`allang[@allang](/users/allang)-pc:~/podman_testing$ podman pull <registry>:5000/pulse/pulse/build:centos-7
Trying to pull <registry>:5000/pulse/pulse/build:centos-7...
Getting image source signatures
Copying blob 5984db63e560 done   |
Copying blob 2d473b07cdd5 done   |
Copying config ff4e77eafb done   |
Writing manifest to image destination
ff4e77eafbab8425e791f1671f14544585a9d2cbdece94603a6cf386afdf0668`

**Reply by nalind** [reply]
As @rhatdan suggested, it appears that one of these is happening:
* you are not logged in to the registry at `docker.io`
* you are logged in to the registry at `docker.io`, but not authorized to access the `docker.io/library/quasar_frontend` repository
* the repository does not exist
